spec/support/shared_examples/queries.rb

```diff
@@ -1,5 +1,6 @@
 # frozen_string_literal: true
 
+require "support/custom_object_changes_adapter"
```

Comment:
Very small nit: probably add a blank line to separate the `require`s from the `RSpec.shared_examples` below?